run.py

```diff
@@ -139,11 +144,21 @@ def home():
             )
 
         word_count = get_word_count(search_string)
-        search_string = get_first_word(search_string)
-
-        # Fetch relevant URLs ordered by page rank
+        words = get_all_words(search_string)
+        words = [i for i in words if i.lower() not in ignored_words]
+        
+        # get list of combined urls
         global orderedURLS
-        orderedURLS = search_db(db_conn=db_conn, word=search_string)
+        orderedURLS = None
+        for word in words:
+            wlist = search_db(db_conn=db_conn, word=word)
+            if orderedURLS is not None: 
+                orderedURLS = list(set(orderedURLS).intersection(wlist))
```

Comment:
Ok so what I'll do is, if there are no common urls I'll get all the links or the Union (rather than intersection) and sort it by rank.
We can reason our implementation in the report.